docs/Roadmap.md

```diff
@@ -0,0 +1,150 @@
+# ONNX Runtime Roadmap
+ONNX Runtime is an active, fast-paced project backed by a strong team of deeply technical Microsoft engineers along with a worldwide 
+community of partners and contributors. This roadmap summarizes the pending investments identified by the team to continually grow 
+ONNX Runtime as a robust, versatile, and high performance engine for DNN and traditional ML models.
+
+## High Level Goals
+ONNX Runtime fully aligns with the goals of the [ONNX](https://onnx.ai/) community to bring openness and interoperability to the AI industry. 
+ONNX is a standardization specification, and ONNX Runtime is an inference engine for models in the ONNX format to efficiently productionize models 
+with high performance.
+
+We recognize the challenges involved in operationalizing ML models performantly in an agile way, and we understand that high volume production services can be highly performance-sensitive and often need to support a variety of compute targets (we experience these first-hand at Microsoft across our vast array of products and services). 
+
+As such, our investments are directly in support of solving those challenges, focusing on areas such as:
```

Comment:
Backwards compatibility is also one of the key challenges faced by teams when operationalizing models and one of the strongest plus point of ORT. It probably deserves a separate bullet of it's own where we mention not just backwards compatibility w.r.t models, but also of the APIs.